python/ml_croissant/ml_croissant/_src/core/json_ld.py

```diff
@@ -0,0 +1,150 @@
+"""json_ld module."""
+
+import json
+from typing import Any
+
+from ml_croissant._src.core import constants
+
+import rdflib
+
+Json = dict[str, Any]
+
+_ML_COMMONS_PREFIX = str(constants.ML_COMMONS)
+_SCHEMA_ORG_PREFIX = str(constants.SCHEMA_ORG)
+# Mapping for non-trivial conversion:
+_PREFIX_MAP = {
+    "http://mlcommons.org/schema/Field": "field",
+    "http://mlcommons.org/schema/RecordSet": "recordSet",
+}
+
+
+def _make_context():
+    return {
+        "@vocab": "https://schema.org/",
+        "applyTransform": "ml:applyTransform",
+        "data": "ml:data",
+        "dataType": "ml:dataType",
+        "field": "ml:Field",
+        "format": "ml:format",
+        "includes": "ml:includes",
+        "ml": "http://mlcommons.org/schema/",
+        "recordSet": "ml:RecordSet",
+        "references": "ml:references",
+        "regex": "ml:regex",
+        "sc": "https://schema.org/",
+        "separator": "ml:separator",
+        "source": "ml:source",
+        "subField": "ml:SubField",
+        "wd": "https://www.wikidata.org/wiki/",
+    }
+
+
+def _is_dataset_node(node: Json) -> bool:
+    """Checks if the type of a node is schema.org/Dataset/"""
+    return node.get("@type") == [str(constants.SCHEMA_ORG_DATASET)]
+
+
+def _sort_items(json_ld: Json) -> list[tuple[str, Any]]:
```

Comment:
It might be confusing to have a variable name be the same as module name, can you use another name?

Reply:
Done.